tests/server/server-executeQuery_error-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+MongoDB\Driver\Server::executeQuery() requires a read preference for secondary
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$rp = new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY);
+$secondary = $manager->selectServer($rp);
+
+echo throws(function() use ($secondary) {
+    $secondary->executeQuery(NS, new MongoDB\Driver\Query(array("x" => 1)));
+}, "MongoDB\Driver\Exception\RuntimeException"), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+OK: Got MongoDB\Driver\Exception\RuntimeException
+not master and slaveOk=false
```

Comment:
I since changed this test to use a conflicting read preference as you requested in an earlier comment. And per the SS spec, the primary read preference should still trigger a "not master and slaveOk=false" error on the secondary, which means this test should still work once [CDRIVER-829](https://jira.mongodb.org/browse/CDRIVER-829) is implemented.